helm_deploy/prisoner-content-hub-frontend/templates/deployment.yaml

```diff
@@ -6,6 +6,11 @@ metadata:
     {{- include "prisoner-content-hub-frontend.labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.replicaCount }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 100%
+      maxUnavailable: 50%
```

Comment:
adding rolling update to prevent downtime when releasing

Reply:
Suggest setting `maxUnavailable: 0`  - so you don't drop down to half number of pods during upgrades.